apps/api/src/services/intent-parser.ts

```
@@ -1,497 +1,464 @@
-  'urgência',
-  'emergência',
-];
-
-export class IntentParser {
-  private contextCache: Map<string, any> = new Map();
+export class IntentParserService {
+  private readonly INTENT_PATTERNS: Record<QueryIntent, RegExp[]> = {
+    client_data: [
+      /(?:clientes?|pacientes?|paciente|client|patients?)\b/i,
+      /(?:most(?:re|ar)?|mostr(?:ar)?|ver|show|list(?:ar)?)\s+(?:todos?\s+)?(?:os\s+)?(?:clientes?|pacientes?)/i,
+      /(?:dados\s+do\s+)?(?:cliente|paciente)\s+([^]+)/i,
```

Biome (2.1.2)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)